rust/src/composepost.rs

```diff
@@ -182,14 +182,17 @@ pub fn compose_postprocess_rpm_macro(rootfs_dfd: i32) -> CxxResult<()> {
 /// Ensure our own `_dbpath` macro exists in the tree.
 #[context("Writing _dbpath RPM macro")]
 fn postprocess_rpm_macro(rootfs_dfd: &openat::Dir) -> Result<()> {
-    let rpm_macros_dir = "usr/lib/rpm/macros.d";
-    rootfs_dfd.ensure_dir_all(rpm_macros_dir, 0o755)?;
-    let rpm_macros_dfd = rootfs_dfd.sub_dir(rpm_macros_dir)?;
-    rpm_macros_dfd.write_file_with("macros.rpm-ostree", 0o644, |w| -> Result<()> {
+    static RPM_MACROS_DIR: &str = "usr/lib/rpm/macros.d";
+    static MACRO_FILENAME: &str = "macros.rpm-ostree";
+    rootfs_dfd.ensure_dir_all(RPM_MACROS_DIR, 0o755)?;
+    let rpm_macros_dfd = rootfs_dfd.sub_dir(RPM_MACROS_DIR)?;
+    rpm_macros_dfd.write_file_with_sync(&MACRO_FILENAME, 0o644, |w| -> Result<()> {
```

Comment:
Though why did this change to `_with_sync()`?  I don't think we need to explicitly fsync here at build time.

Reply:
Ack, that was a bit overkill, dropped.